Documentation/operations/upgrade.rst

```diff
@@ -226,7 +224,12 @@ version which was installed in this cluster. Valid options are:
    .. code-block:: yaml
 
       agent: true
-      upgradeCompatibility: 1.7
+      upgradeCompatibility: "1.7"
```

Comment:
```suggestion
      upgradeCompatibility: "1.8"
```

Otherwise, the options are still not the same :). (Or maybe even bump to 1.9 everywhere?)

Reply:
yeah I'll think better to just bump to 1.8. We can revise whether these extra notes are necessary as we approach the 1.10 release window.